lxd/devices.go

```diff
@@ -317,6 +318,20 @@ func deviceLoadGpu(all bool) ([]gpuDevice, []nvidiaGpuDevice, error) {
 			}
 		}
 
+		// Retrieve node ID
+		numaPath := fmt.Sprintf(filepath.Join(device, "numa_node"))
+		numaNode := uint64(0)
+		if shared.PathExists(numaPath) {
+			numaID, err := shared.ParseNumberFromFile(numaPath)
+			if err != nil {
+				continue
```

Comment:
You might want to log a warning in this case.

Reply:
We probably should but we haven't done so for any of the other files, so going with consistency for now :)

This entire code will need to go away soon and be replaced by something quite a bit more robust.